en/docs/migrate-to-choreo/why-migrate-from-api-cloud-to-choreo.md

```diff
@@ -0,0 +1,20 @@
+# Why Migrate From API Cloud to Choreo?
+
+Choreo, the latest fully-fledged cloud offering by WSO2, is a digital innovation platform that allows you to develop, deploy, and manage cloud-native applications at scale. For more details, see [Choreo documentation](https://wso2.com/choreo/docs/).
+
+## Enhanced user experience in API management capabilities with Choreo
+Full lifecycle API management is a key capability of Choreo.
+
+-  The Choreo Console (equivalent to the API Publisher in API Cloud) allows API developers to design, publish, and manage the lifecycle of an API.
+-  The API Developer Portal (equivalent to the API Store in API Cloud) allows consumers to consume APIs. 
+
+Overall, Choreo provides users with an improved user experience in API management.
+
+## End of life of WSO2 API Cloud
+WSO2 API Cloud was the WSO2 SaaS platform for API Management from 2015 onwards. Now, with the introduction of Choreo, the operations of WSO2 API Cloud comes to an end. The deadline for the end of life of API Cloud is December 31, 2023.
+
+## A futuristic digital innovation platform for API Cloud customers
```

Comment:
The subject may not be suitable, the purpose of this section is to let customers know what are the options they have when they are asked to move away from API Cloud, what should be their future direction within WSO2.

Reply:
Updated to "Future of WSO2 API Cloud customers" to align with the original subheading in the gdoc.